packages/global/start-server.js

```diff
@@ -22,10 +22,10 @@ const defaultContentGatingHandler = require('./utils/content-gating-handler');
 const routes = (siteRoutes, siteConfig) => (app) => {
   // Handle submissions on /__inquiry
   loadInquiry(app);
-  // Shared/global routes (all sites)
-  sharedRoutes(app, siteConfig);
   // Load site routes
   siteRoutes(app);
+  // Shared/global routes (all sites)
+  sharedRoutes(app, siteConfig);
```

Comment:
@Shinsina - on one hand, I can see why you changed the order.  If the site routes run first, it allows for a custom /podcast route (and therefore template) to override the global version.  Makes sense...except that this customer is now different than everyone else, which we don't want.

So...one of two things needs to happen:
- Discuss as a group and make this the new standard across all customers.  Would likely break a lot of things that expect the reverse order.
- Take the /podcasts route out of the Global package for PMMI and add to each site individually.  All sites except AW can still reference the global template, but AW can reference the custom site one.

#2 is far less invasive and probably the way we should go for now, but I'm open to discussing.